docs/release-notes/cloud/cloud-2022-02-16-pmd-legacy-removal.md

```diff
@@ -0,0 +1,16 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Removal of PMD (Legacy) February 16, 2022
+
+On the week of February 16th 2022 we'll be removing the tool **PMD (Legacy)** from Codacy.
```

Comment:
```suggestion
On the week of February 16th 2022 Codacy will stop supporting the **PMD (Legacy)** linter.
```

Reply:
dunno if linter is better than tool though, probably @prcr will be able to help here

Reply:
Throughout the docs we consistently refer to "tools", for example:

- https://docs.codacy.com/getting-started/supported-languages-and-tools/
- https://docs.codacy.com/related-tools/local-analysis/client-side-tools/

Would you say that "linter" is more correct, or more specific perhaps? We can always consider adjusting the wording used on the docs, although I would do it as a separate task. :slightly_smiling_face: